arangopipe/arangopipe_frontend/app/src/containers/App/App.jsx

```diff
@@ -55,6 +56,7 @@ class App extends React.Component{
      } else {
        this.getCurrentUser()
      }
+     console.log(this.props)
```

Comment:
Stray console.log here or is this intentional?

Reply:
Query functionality is in the middle of developing now. So don't pay attention to the code now. After finishing, let me clean the code.